cmd/neofs-node/control.go

```diff
@@ -70,6 +58,17 @@ func initControlService(c *cfg) {
 			c.wg.Done()
 		})
 	}()
+
+	c.workers = append(c.workers, newWorkerFromFunc(func(ctx context.Context) {
```

Comment:
why async worker? U know exactly when all needed components are ready

Reply:
think that is our common pattern but it was about IR, fixed